docs/_themes/clusterhq/layout_homepage.html

```diff
@@ -43,20 +43,20 @@
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
           </button>
-          <a class="navbar-brand" href="#"><img src="{{ pathto('_static/images/', 1) }}logo@2x.png" alt="ClusterHQ Container Data Management" /></a>
+          <a class="navbar-brand" href="https://clusterhq.com/"><img src="{{ pathto('_static/images/', 1) }}logo@2x.png" alt="ClusterHQ Container Data Management" /></a>
```

Comment:
Why introduce an absolute link here?  If someone is looking at the docs on dev.clusterhq.com then I guess this is the wrong link.  `/` seems like it might be better (though it's still wrong if you're looking at a `file://` URI but I dunno how to fix that).